src/app/models.py

```diff
@@ -27,6 +27,29 @@ class Role(str, Enum):
     USER = "user"
 
 
+class AnnotationType(str, Enum):
```

Comment:
Do we want to have different types of smoke: wildfire, industrial, chimney, agriculture, etc or also all the FP types?